lib/oelite/fetch/git.py

```diff
@@ -267,6 +267,17 @@ def unpack(self, d):
         return True
 
     def mirror(self, mirror=os.getcwd()):
+        if self.is_local:
+            # Don't mirror if the source is local. Concept taken from fetch
+            # Do check revision exists before accepting though
+            print "Warning: Skipping creating mirror for local path %s"%(self.uri)
```

Comment:
In the line of "least surprises" it might be nice to be told that stuff is not being mirrored, whether it should say warning *shrug* :)